bin/start_kube.sh

```diff
@@ -253,6 +253,21 @@ else
   pulumi config set prometheus:adminpass -C ${script_dir}/../pulumi/python/config
 fi
 
+#
+# TODO: Allow startup scripts to prompt and accept additional config values #97
+# The default helm timeout for all of the projects is set at the default of 300 seconds (5 minutes)
+# However, since this code path is most commonly going to be used to deploy locally we need to bump
+# that value up. A fix down the road will add this a prompt, but for now we are going to double this
+# value for all helm deploys.
+#
+
+pulumi config set kic-helm:helm_timeout 600 -C ${script_dir}/../pulumi/python/config
```

Comment:
Do you want to use the default timeout value you established previously?

Reply:
Yes - I want that to be the default for all of the cloud-y deploys like AWS, and once we merge the code I want to let the user pick. For now though, some of the deploys take between 6 and 8 minutes on these smaller VM's (depending on the speed and PV speed). So, this is entirely a work-around that can be reverted once we fix the startup script, but it will reduce pain if users try and follow our microk8s install instructions.